Cargo.toml

```diff
@@ -52,3 +52,6 @@ slog-async = "2"
 
 [package.metadata.docs.rs]
 features = ["std", "nested-values", "dynamic-keys"]
+
+[patch.crates-io]
```

Comment:
This will have to go if this is to be landed.

Reply:
Sure, I know, just wanted to make whole code build.